spec/licensee/projects/git_hub_project_spec.rb

```diff
@@ -109,4 +109,20 @@
       expect { subject.license }.to raise_error(described_class::RepoNotFound)
     end
   end
+
```

Comment:
Do the tests below test that licensee can successfully detect a license on the ref provided, and not eg a different license on the default branch? Not certain needed, just seems an obvious question.

Reply:
I thought about the helpfulness of writing that test since most of the test will be mocked behavior of github APIs. 
Although I think it would be helpful to add the test to assert the request we're stubbing correctly contains the query params. I'll add this in along with the cli doc changes 👍